### ISSUE TITLE
Droplets disappear with multiple meshes

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#294]
```
Please complete the following lines...

Application Version: 5.1.4 Parallel
SVN Revision Number:1437
Compile Date: 14 March 2008
Operating System: Linux

Describe details of the issue below:

The attached case demonstrates three issues with droplets and multiple 
meshes.

Nozzle 1: droplets are lost when on a surface and crossing mesh boundary 
in y-direction

Nozzle 2: droplets are lost when on a surface and crossing mesh boundary 
in z-direction

Nozzle 3: droplets only appear in one quadrant when nozzle placed where 
four meshes meet

Regards,

Jukka Vaari

```

Original issue reported on code.google.com by `jukka.vaari@vtt.fi` on 2008-03-28 14:51:02

<hr>
- _Attachment: [spray.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-315/comment-0/spray.fds)_

**Comment by gforney** [comment]
```
For 3, currently devices are assigned to a single mesh as it makes the MPI stuff 
much easier to deal with.  So for now don't put a nozzle where it is at a mesh 
boundary or close enough to the boundary that adding OFFSET to the nozzle location

will wind up in another mesh.
```

Original issue reported on code.google.com by `drjfloyd` on 2008-03-28 15:22:30
- **Labels added**: Application-FDS, OpSys-All

**Comment by gforney** [comment]
```
The same defect happens by running spray.fds with 5.1.4 serial.
Riccardo Sala
```

Original issue reported on code.google.com by `salarik@libero.it` on 2008-03-29 15:41:47

**Comment by gforney** [comment]
```
Nice test case. The fact that the droplets pass properly from one mesh to another 
only in certain directions is a big hint. I should be able to fix this. Jason is 
right about number 3 -- we should add a warning about this. 
```

Original issue reported on code.google.com by `mcgratta` on 2008-04-02 14:00:35

**Comment by gforney** [comment]
```
For what it's worth, here's another little test case demonstrating rain at mesh 
boundary.
```

Original issue reported on code.google.com by `jukka.vaari@vtt.fi` on 2008-04-02 14:17:34

<hr>
- _Attachment: [mist.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-315/comment-4/mist.fds)_

**Comment by gforney** [comment]
```
I fixed the droplet crossing mesh boundary bug, SVN 1533 and higher. Update your 
source or wait for the next release, then try the spray case again.
```

Original issue reported on code.google.com by `mcgratta` on 2008-04-03 16:32:00

**Comment by gforney** [comment]
```
Works fine now. Thanks !

Jukka
```

Original issue reported on code.google.com by `jukka.vaari@vtt.fi` on 2008-04-03 18:21:44

**Comment by gforney** [comment]
```
I'll mark as verified, and it will be included in FDS 5.1.5
```

Original issue reported on code.google.com by `mcgratta` on 2008-04-03 20:32:53
